k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api
+spec:
+  releaseName: ccpay-refunds-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
```

Comment:
Repository is incorrect. It should point to charts in Github.

Example: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml#L8-L10